python-lib/ontology_tagger.py

```diff
@@ -25,7 +31,7 @@ class Tagger:
             Used if there is only one language to treat.
             Use the argument 'language_column' for passing a language column name in 'tag_and_format' method otherwise.
         lemmatization (bool): If True , match on lemmatized forms. Default is False.
-        case_insensitivity (bool): If True, match on lowercased forms. Default is False.
+        normalize_case (bool): If True, match on lowercased forms. Default is False.
```

Comment:
I would not use the word `normalize` here. Rather something like `lowercase`, simply

Reply:
'lowercase' may indeed be a better word. We decided to use 'normalize_case' to align with the plugin UI and with the other NLP plugin UIs :)